Automattic/vip-go-mu-plugins
# adds index name to logs

## Description
Adds more info into a log messgae

## Changelog Description

### Plugin Updated: Search

Adds more information into helthcheck failure log.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Make sure validate couns will brek by adding: `$result = new WP_Error('Failed again');` `search/includes/classes/class-health.php:235`
1) print error msg by adding: `var_dump('msg:', $message);` in `search/includes/classes/class-healthjob.php:238`
1) in `wp shell`:
```
$s = new \Automattic\VIP\Search\Search();
$s->init();
$hj = new \Automattic\VIP\Search\HealthJob($s);
$hj->check_document_count_health();
```
